handlers/admin/data-contributions.js

```diff
@@ -16,23 +20,30 @@ function editDataContribution(request, reply) {
       request.yar.flash('success', 'Data Contribution updated successfully.');
       return reply.redirect('/admin/data-contributions');
     })
-    .catch(() => {
-      // FIXME: Log error
+    .catch((error) => {
+      console.error(error.detail); // eslint-disable-line no-console
```

Comment:
Can we have something like... a log file? I'm generally not happy having `console.whatever()` statements in the code, due to their usual `STDOUT`/`STDERR` output. While that's not inherently bad, it might be harder to observe in our case.
Also, would it make sense for us to create a "standardise logging throughout the whole app" issue? So we'd send things to a global logger that pipes the output through to various places.